README.md

```diff
@@ -2,26 +2,26 @@
 
 A collection of packages to augment `testing` and support common patterns.
 
-[![GoDoc](https://godoc.org/gotest.tools?status.svg)](https://godoc.org/gotest.tools)
+[![GoDoc](https://godoc.org/gotest.tools?status.svg)](http://gotest.tools)
```

Comment:
https ? :angel:

Reply:
I guess that works too, I wasn't sure. The url doesn't have any content and redirects immediately to https://godoc.org/gotest.tools/